wp-content/themes/citylimits/functions.php

```diff
@@ -498,3 +511,27 @@ function zonein_tax_archive_query( $query ) {
 	}
 }
 add_action( 'pre_get_posts', 'zonein_tax_archive_query', 1 );
+
+/* Custom query for event list widget*/
```

Comment:
This PR should included docs somewhere explaining why the event list widget _only_ displays stuff from the Communitywire Events area.

Eventually, ideally, we'd find a better way to do this filter. But it's not necessary now.